.github/workflows/go.yml

```diff
@@ -28,13 +28,15 @@ jobs:
       run: |
         go install github.com/playwright-community/playwright-go/cmd/playwright
         playwright install --with-deps
-        go test -v ./... -coverprofile=coverage.txt -covermode=atomic
+        go test -v ./... -coverprofile=${{ matrix.os }}_coverage.txt -covermode=atomic
 
-    - name: Codecov
-      uses: codecov/codecov-action@v3
+    - name: Upload coverage report
```

Comment:
@briantist please add below after this line

```
if: matrix.os == 'ubuntu-18.04'
```

Reply:
@briantist thank you. I proposed as it corrected Codecov reporting in my setup. Do reports have unique names (based on OS)?

Reply:
I suppose based on how they are being uploaded, yes! 
`go test -v ./... -coverprofile=${{ matrix.os }}_coverage.txt -covermode=atomic`

Reply:
@mapkon @briantist I didn't catch that update. thx